js/foundation.dropdown.js

```diff
@@ -51,6 +51,11 @@ class Dropdown {
 
     });
 
+    if(this.options.parentClass){
+      this.$parent = $('.' + this.options.parentClass);
```

Comment:
What will happen if there are multiple examples of the parent class?  Should this search up this.$element.parents() ?